src/transformers/modeling_utils.py

```diff
@@ -2188,6 +2190,17 @@ def from_pretrained(
         subfolder = kwargs.pop("subfolder", "")
         commit_hash = kwargs.pop("_commit_hash", None)
         variant = kwargs.pop("variant", None)
+        peft_adapter_name = kwargs.pop("peft_adapter_name", "default")
+        peft_adapter_model_id = kwargs.pop("_peft_adapter_model_id", None)
+
+        has_adapter_file = False
+        if is_peft_available():
+            has_adapter_file = peft_adapter_model_id is not None
+
+        if has_adapter_file:
+            logger.info(
+                f"Found adapter file at {peft_adapter_model_id}. Automatically loading adapter model using the base model from {pretrained_model_name_or_path}"
+            )
```

Comment:
Need to be split on several lines.

Reply:
Fixed in cd22c543ef06b336dc5735a9fdc149c87c20d9f3